### PR TITLE
Remove 'be formatted' from VolumeSource docs

### PR DESCRIPTION
**Pull request by ghost**
Volumes do not need to be formatted as we now have the
SafeFormatAndMount utility which formats volumes when needed.

### CODE REVIEW

**Comment by swagiaal** [comment]
@bgrant0607 PTAL

File: pkg/api/types.go [hunk 1]
```diff
@@ -441,10 +441,10 @@ const (
 
 // Represents a Persistent Disk resource in Google Compute Engine.
 //
-// A GCE PD must exist and be formatted before mounting to a container.
-// The disk must also be in the same GCE project and zone as the kubelet.
-// A GCE PD can only be mounted as read/write once.
-// GCE PDs support ownership management and SELinux relabeling.
+// A GCE PD must exist before mounting to a container. The disk must
+// also be in the same GCE project and zone as the kubelet. A GCE PD
+// can only be mounted as read/write once. GCE PDs support ownership
```
Comment (saad-ali) [inline pkg/api/types.go]: > `can only be mounted as read/write once`

While, we're cleaning up the comments, this doesn't seem right. GCE PDs can be mounted as read-only as well right? Ditto for AWS below?

**Comment by swagiaal** [comment]
@saad-ali updated GCE comment. I think AWS EBS is correct

**Comment by saad-ali** [comment]
LGTM thanks @swagiaal

**Comment by ixdy** [comment]
@faiboy this was failing verification checks and should not have been merged. `kubernetes-test-go` is now broken as a result.

**Comment by ixdy** [comment]
@fabioy this was failing verification checks and should not have been merged. `kubernetes-test-go` is now broken as a result.

**Comment by fabioy** [comment]
Yeah, I missed it (my bad). I've since updated docs on another PR.

**Comment by swagiaal** [comment]
Hmm.. sorry about that guys!
Which verification test was failing ?
I can only see
```
[Fail] Deployment [It] RollingUpdateDeployment should delete old pods and create new ones
```

**Comment by fabioy** [comment]
Yeah, it's not in the "Build Log" link above, but the Jenkins output. But, if you build locally, it'll show an error in the generated doc:

Moving api reference docs from /workspace/kubernetes/_output/generated_html to /workspace/kubernetes/_tmp/api-reference
diffing /workspace/kubernetes/docs/api-reference against freshly generated docs
diff -NauprB -I 'Last update' '--exclude=*.md' /workspace/kubernetes/docs/api-reference/extensions/v1beta1/definitions.html /workspace/kubernetes/_tmp/api-reference/extensions/v1beta1/definitions.html
--- /workspace/kubernetes/docs/api-reference/extensions/v1beta1/definitions.html	2016-02-26 13:34:34.047480330 -0800
+++ /workspace/kubernetes/_tmp/api-reference/extensions/v1beta1/definitions.html	2016-02-26 13:38:35.811611144 -0800
@@ -1437,7 +1437,7 @@ Examples:<br>
 <p>Represents a Persistent Disk resource in Google Compute Engine.</p>
 </div>
 <div class="paragraph">
-<p>A GCE PD must exist before mounting to a container. The disk must also be in the same GCE project and zone as the kubelet. A GCE PD can only be mounted as read/write once. GCE PDs support ownership management and SELinux relabeling.</p>
+<p>A GCE PD must exist before mounting to a container. The disk must also be in the same GCE project and zone as the kubelet. A GCE PD can only be mounted as read/write once or read-only many times. GCE PDs support ownership management and SELinux relabeling.</p>
 </div>
 <table class="tableblock frame-all grid-all" style="width:100%; ">
 <colgroup>
diff -NauprB -I 'Last update' '--exclude=*.md' /workspace/kubernetes/docs/api-reference/v1/definitions.html /workspace/kubernetes/_tmp/api-reference/v1/definitions.html
--- /workspace/kubernetes/docs/api-reference/v1/definitions.html	2016-02-26 13:34:34.047480330 -0800
+++ /workspace/kubernetes/_tmp/api-reference/v1/definitions.html	2016-02-26 13:38:28.151607127 -0800
@@ -982,7 +982,7 @@ Examples:<br>
 <p>Represents a Persistent Disk resource in Google Compute Engine.</p>
 </div>
 <div class="paragraph">
-<p>A GCE PD must exist before mounting to a container. The disk must also be in the same GCE project and zone as the kubelet. A GCE PD can only be mounted as read/write once. GCE PDs support ownership management and SELinux relabeling.</p>
+<p>A GCE PD must exist before mounting to a container. The disk must also be in the same GCE project and zone as the kubelet. A GCE PD can only be mounted as read/write once or read-only many times. GCE PDs support ownership management and SELinux relabeling.</p>
 </div>
 <table class="tableblock frame-all grid-all" style="width:100%; ">
 <colgroup>
/workspace/kubernetes/docs/api-reference is out of date. Please run hack/update-api-reference-docs.sh
FAILED

**Pull request merged**